packages/x-svg-converter/src/__tests__/svg-to-vue.spec.ts

```diff
@@ -0,0 +1,102 @@
+import path from 'path';
+import fs from 'fs';
+import { svgToVue } from '../svg-to-vue';
+import svgStub from './svg-stub';
+
+// Mock to prevent the prettier from running in test env.
+jest.mock('child_process', () => {
```

Comment:
What do you think about adding a test to check if `prettier` is being called after creating the files?